config/routes.rb

```diff
@@ -1,4 +1,9 @@
 Rails.application.routes.draw do
   root to: 'pages#home'
+  resources :users, except [:index, :new, :create]
+  resources :goats do
+    resources :reserves
```

Comment:
Except edit, update and destroy, those should not be nested (it does not depend on the goats_id)